.github/templates/metagraph_currency_l1/initial_validator/action.yml

```diff
@@ -30,7 +30,7 @@ runs:
       shell: bash
       run: |
         cd .github/templates/project_template/
-        sbt currencyL1/assembly
+        sbt -Dversion.override=99.99.99 currencyL1/assembly
```

Comment:
You can remove the 
-Dversion.override=99.99.99
it's unnecessary here

Reply:
:+1: